### DISCUSSION TITLE
CH03_06 AWS 계정 SSH Key 등록 및 EC2 접속방법  중 접속이 안됩니다.

### DISCUSSION THREAD

**Discussion by jhdl0157** [tedilabs/fastcampus-devops#26] [Q&A] [answered]
![dhfb](https://user-images.githubusercontent.com/72914519/140003237-d10b0ac5-ec04-4d2c-8857-33eed6665158.png)

위와 같이 chmod가 안듣는거 같은데 어떻게 해야하나요??

**Comment by jhdl0157** [comment, marked as answer]
https://techsoda.net/windows10-pem-file-permission-settings/ 이 블로그 를 보고 따라서 해봤는데 연결은 했습니다.
근데 ls -l을 찍으면 -r-xr-xr-x 1 root root 1678 Nov  3 11:09 fastcampus.pem이런식으로 나옵니다.
나중에 실습을 할때 문제가 있을까요??

**Reply by posquit0** [reply]
안녕하세요. 현재 작업을 하고 있는 디렉토리의 위치를 보면 `/mnt/c`로 윈도우 호스트 디렉토리입니다.
우분투 운영체제에서는 우분투의 기본 파일시스템 내에 위치한 디렉토리에서 작업을 하는 것이 해당 권한 이슈를 마주치지 않을 것 같습니다.
본인의 홈 디렉토리 `~/` 내에서 pem 키를 다운로드 받아 권한 설정 후 사용하시면 문제 해결에 도움이 될 것 같습니다.

**Reply by jhdl0157** [reply]
감사합니다 홈디렉토리에서 폴더 만들고 하니 잘 작동 합니다.
